### ISSUE TITLE
Does not work with Webpack imports

### ISSUE THREAD

**Issue by souporserious** [ginpei/html5-youtube.js#20]
It doesn't look like this library works with Webpack for imports 😕 I can put together a PR and try and fix this if you would like? Please let me know.

**Comment by ginpei** [comment]
Right, I didn't care about such tools. Sorry for inconvenient. I'd appreciate if you give me the PR! 😸

**Comment by souporserious** [comment]
No worries, I think it should be a simple fix 🙂, I'll get a PR going as soon as I get a chance.
